Makefile

```diff
@@ -71,9 +71,22 @@ setup: ### Setup remote environment
 	for file in $(PROJECT_FILES); do $(NEURO) cp ./$$file $(PROJECT_PATH_STORAGE)/$$file; done
 	$(NEURO) exec --no-tty --no-key-check $(SETUP_JOB) "bash -c 'export DEBIAN_FRONTEND=noninteractive && $(APT) update && cat $(PROJECT_PATH_ENV)/apt.txt | xargs -I % $(APT) install --no-install-recommends % && $(APT) clean && $(APT) autoremove && rm -rf /var/lib/apt/lists/*'"
 	$(NEURO) exec --no-tty --no-key-check $(SETUP_JOB) "bash -c '$(PIP) -r $(PROJECT_PATH_ENV)/requirements.txt'"
+ifdef __BAKE_SETUP
+	make __bake
+endif
 	$(NEURO) --network-timeout 300 job save $(SETUP_JOB) $(CUSTOM_ENV_NAME)
 	$(NEURO) kill $(SETUP_JOB)
 
+.PHONY: __bake
+__bake: upload-code upload-data upload-notebooks
+	echo "#!/usr/bin/env bash" > jupyter.sh
+	echo "jupyter notebook --no-browser --ip=0.0.0.0 --allow-root \
+	    --NotebookApp.token= --notebook-dir=/project/notebooks" >> jupyter.sh
+	$(NEURO) cp jupyter.sh $(PROJECT_PATH_STORAGE)/jupyter.sh
+	$(NEURO) exec --no-tty --no-key-check $(SETUP_JOB) \
+	    "bash -c 'mkdir -p /project; cp -R -T $(PROJECT_PATH_ENV) /project'"
```

Comment:
and here also, let's not introduce this hack but rather make path '/project' within the job for all Template projects?

Reply:
this could be `/whatever`. I don't think this is related to the scope of this PR.

Reply:
I meant to change it in Tepmlate https://github.com/neuromation/cookiecutter-neuro-project/issues/353
Anyways we are going to release Template v1.4 soon, there's just [Windows integration](https://github.com/neuromation/cookiecutter-neuro-project/pull/350) pending for review